src/parser/ha_parser.py

```
@@ -0,0 +1,191 @@
+
+            # 12. 予約/フラグ領域 (位置:33, 長さ:31)
+            # 16バイトのゼロ + 15バイトのスペース
+            result["Reserved"] = self.decode_field(data[pos : pos + 31])
+            pos += 31
+
+            # pos = 63: 払戻データ開始
+            # 15バイトエントリー: Kumi(2) + Pay(13)
+            entries = self._parse_payout_entries(data, pos)
+
+            # エントリーを結果辞書に格納
+            # 通常エントリー（kumi != '00'）と合計行（kumi == '00'）を分離
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)